packages/pyright-internal/src/commands/createTypeStub.ts

```diff
@@ -9,53 +9,64 @@
 import { CancellationToken, ExecuteCommandParams } from 'vscode-languageserver';
 
 import { OperationCanceledException } from '../common/cancellationUtils';
-import { LanguageServerInterface } from '../common/languageServerInterface';
+import { LanguageServerBaseInterface, LanguageServerInterface } from '../common/languageServerInterface';
 import { AnalyzerServiceExecutor } from '../languageService/analyzerServiceExecutor';
 import { ServerCommand } from './commandController';
 import { Uri } from '../common/uri/uri';
+import { Workspace } from '../workspaceFactory';
 
 export class CreateTypeStubCommand implements ServerCommand {
-    constructor(private _ls: LanguageServerInterface) {}
+    constructor(private _ls: LanguageServerInterface) {
+        // Empty
+    }
```

Comment:
this also very strange

Reply:
Probably a hygiene issue?

Reply:
another bad merge

Reply:
actually it was a change in pylance, some formatting still seems strange though